### PR TITLE
Profile Page

### PR DESCRIPTION
**Pull request by raju-max**

### CODE REVIEW

File: src/components/AdvancedSettings.vue [hunk 1]
```diff
@@ -58,7 +58,15 @@
       </a>
       <div class="collapse" id="panel-11">
         <div class="pt-3">
-          <div class="form-group mb-4 focused">
+          <div class="form-group">
+                    <label class="my-1 mr-2" for="inlineFormCustomSelectPref">Country</label>
+                    <select class="custom-select my-1 mr-sm-2" id="inlineFormCustomSelectPref">
+                        <option selected="">Choose...</option>
```
Comment (chayan007) [inline src/components/AdvancedSettings.vue:64]: Please use -> `data/country.json`, which will have the following:

```
{
"IN": "India",
"US": "United States of America",
"CA": "Canada",
...
}
```

File: src/components/CategoryForm.vue [hunk 2]
```diff
@@ -0,0 +1,100 @@
+<template>
+    
+        <div class="">
+                
+                    <div class="a-1 space scroll">
+                <!--Buttons-->
+                
+                <!--End of Buttons-->
```
Comment (chayan007) [inline src/components/CategoryForm.vue:8]: remove unnecessary comments.

File: src/components/PostButton.vue [hunk 3]
```diff
@@ -0,0 +1,56 @@
+<template>
+    <div class="cant bg-primary shadow-soft">
+        <button type="submit" class="a1 btn btn-pill btn-primary">
+            <img src="../assets/Edit.png" alt="sign in" width="30">
```
Comment (chayan007) [inline src/components/PostButton.vue:4]: `alt` to be changed for all of the buttons.

File: src/services/user.service.js [hunk 4]
```diff
@@ -12,3 +13,20 @@ function getDetails() {
     };
     return fetch(`${config.commonConfig.$apiUrl}/users`, requestOptions).then(handleResponse);
 }
+/*function edit_settings(settings_data){
```
Comment (chayan007) [inline src/services/user.service.js:16]: remove if not required.

File: src/components/ProfileMenu.vue [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+    z-index: 299;
+    background-color: rgba(0,0,0,0.2);
+
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    
+}
+.popup-inner{
+
```
Comment (chayan007) [inline src/components/ProfileMenu.vue:41]: please remove spaces.

File: src/components/SettingsPage.vue [hunk 6]
```diff
@@ -84,7 +85,7 @@
             ><span
               class="icon-title h6 mb-0 font-weight-bold"
               data-v-6db336fc=""
-              >Change <a style="color:green">Profile Password</a></span
+              >Change <a style="color: green">Profile Password</a></span
```
Comment (chayan007) [inline src/components/SettingsPage.vue:88]: reason for adding a space ?

File: src/configurations/user.js [hunk 7]
```diff
@@ -8,7 +8,10 @@ if (process.env.NODE_ENV === "PRODUCTION") {
             },
             login: {
                 "endpoint": "profile/auth/login/"
-            }
+            },
```
Comment (chayan007) [inline src/configurations/user.js:11]: delete if not required.

File: src/components/LoginForm.vue [hunk 8]
```diff
@@ -123,9 +123,7 @@ export default {
   name: "LoginForm",
   data () {
     return {
-      username: '',
-      password: '',
-      submitted: false
+
```
Comment (chayan007) [inline src/components/LoginForm.vue:126]: Why are you removing these values ? This will stop the login flow ?

**Review by chayan007 — CHANGES_REQUESTED** [review]

**Review by itisdb — APPROVED** [review]

**Review by chayan007 — APPROVED** [review]

File: src/views/Category.vue [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+    </div>
+</template>
+
+<script>
+// @ is an alias to /src
+import CategoryForm from '@/components/CategoryForm.vue'
+import PeopleCard from '@/components/PeopleCard.vue'
+export default {
+  name: 'Category',
+  components: { CategoryForm ,PeopleCard
```
Comment (chayan007) [inline src/views/Category.vue:27]: please check this kind of irregularities.

File: src/views/Post.vue [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+</template>
+
+<script>
+// @ is an alias to /src
+import Post1 from "@/components/Post1.vue";
+import PostButton from "@/components/PostButton.vue";
+import PostButton2 from "@/components/PostButton2.vue";
+export default {
+  name: "Post",
+  components: { Post1,PostButton,PostButton2 },
```
Comment (chayan007) [inline src/views/Post.vue:20]: give one space after `,`

File: src/views/SingleChatPage.vue [hunk 11]
```diff
@@ -0,0 +1,26 @@
...
+  </div>
+</template>
+
+<script>
+// @ is an alias to /src
+
+import SingleChat from "@/components/SingleChat.vue";
+export default {
+  name: "Chat",
+  components: { SingleChat, },
```
Comment (chayan007) [inline src/views/SingleChatPage.vue]: why is there a trailing `,` for no reason ?

**Review by chayan007 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**